packages/ai-ide/src/browser/task-context-file-storage-service.ts

```diff
@@ -41,8 +41,7 @@ export class TaskContextFileStorageService implements TaskContextStorageService
         return label.replace(/^[^\p{L}\p{N}]+/vg, '');
     }
 
-    protected async getStorageLocation(): Promise<URI | undefined> {
-        await this.workspaceService.ready;
+    protected getStorageLocation(): URI | undefined {
```

Comment:
The `store` function also calls `getStorageLocation` and therefore now no longer waits until the workspaceService is ready. Also it still `await`s the `getStorageLocation` call.

To fully restore the old behavior, `store` should await the `workspaceService.ready`.